### PR TITLE
Animals table

### PR DESCRIPTION
**Pull request by smunoz1988**
Hello! 

for this milestone, I make this changes:

- Create a database named vet_clinic.
- Create a table named animals with the following columns:

          1. id: integer
          2. name: string
          3. date_of_birth: date
          4. escape_attempts: integer
          5. neutered: boolean
          6. weight_kg: decimal
        
- Insert the required data to the table.
- Write queries for the following quests:

          1. Find all animals whose name ends in "mon".
          2. List the name of all animals born between 2016 and 2019.
          3. List the name of all animals that are neutered and have less than 3 escape attempts.
          4. List the date of birth of all animals named either "Agumon" or "Pikachu".
          5. List name and escape attempts of animals that weigh more than 10.5kg
          6. Find all animals that are neutered.
          8. Find all animals not named Gabumon.
          9. Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,191 +52,93 @@ After you're finished please remove all the comments and instructions!
...
+- Twitter: [@twitterhandle](https://twitter.com/Santiag24209785)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/santiago-munoz-0b2b1a260)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FUTURE FEATURES -->
-
 ## 🔭 Future Features <a name="future-features"></a>
 
 > Describe 1 - 3 features you will add to the project.
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Kindly delete this instruction

File: README.md [hunk 2]
```diff
@@ -96,191 +52,93 @@ After you're finished please remove all the comments and instructions!
...
+- **[Database for vet animals]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+Not able at the moment
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] As you don't have a link yet, kindly delete this section

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @smunoz1988 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴 

You have submitted a project, that has some invalidation sections.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 3]
```diff
@@ -96,191 +51,84 @@ After you're finished please remove all the comments and instructions!
...
-
-- **[Question_2]**
-
-  - [Answer_2]
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
```
Comment (omar-muhamad) [inline README.md:134]: - [ ] The MIT link is broken, Please add a valid link to the LICENSE file.

image

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @smunoz1988,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @smunoz1988,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**